bin/install-rpicam-mjpeg.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -e
+
+LIBCAMERA_REMOTE=https://github.com/raspberrypi/libcamera.git
+RPICAM_APPS_REMOTE=https://github.com/Windermere-Technology/rpicam-mjpeg.git
+
+# Compile and install libcamera from source
+install_libcamera() {
+    set -e
+    git clone "$LIBCAMERA_REMOTE" libcamera
```

Comment:
```suggestion
    # install via apt if available
    if sudo apt install -y libcamera-dev; then
        return
    fi

    git clone "$LIBCAMERA_REMOTE" libcamera
```
I guess development headers are needed for the `rpicam-mjpeg` build, otherwise could be adjusted.

Reply:
Is the version in apt updated now? When we were doing development the repos were too outdated to actually compile the latest `rpicam-*`

Reply:
Which repos in particular? `archive.raspberrypi.com` ships latest libcamera 0.5.1, and is usually kept up-to-date tightly. The Debian (and Ubuntu) repo of course is outdated, naturally since they usually do not update any software version within one Debian release. So in that case the question is whether RPi OS (and others which use `archive.raspberrypi.com` for RPi-specific software) is targeted, or whether this shall work with plain Debian or Ubuntu repo as well.